### DISCUSSION TITLE
CSS class sorting as safe action / config rule as safe or unsafe

### DISCUSSION THREAD

**Discussion by gustavo-bonfim** [biomejs/biome#1714] [General]
It would be great if we had an option in biome.json or in the editor's extensions that allows us to choose which unsafe rules we prefer to fix on save. Currently, the extension only patches the safe rules.

I would appreciate it if CSS classes could be automatically sorted when I save a file, similar to other rules.

Is it necessary for the order of CSS classes to be considered an unsafe rule?

**Comment by DaniGuardiola** [comment]
I support your suggestion to allow opting into unsafe rules, and I would recommend that you reframe this discussion from this angle. The class sorting rule is a work in progress, far from complete, so we don't plan on making it safe for now since it could break code for users.
